src/System.Text.Json/src/System/Text/Json/Node/JsonString.cs

```diff
@@ -0,0 +1,81 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Text.Json
+{
+    /// <summary>
+    ///   Represents a text JSON value.
+    /// </summary>
+    public class JsonString : JsonNode, IEquatable<JsonString>
+    {
+        /// <summary>
+        ///   Initializes a new instance of the <see cref="JsonString"/> class representing the empty value.
+        /// </summary>
+        public JsonString() => Value = "";
+
+        /// <summary>
+        ///   Initializes a new instance of the <see cref="JsonString"/> class representing a specified value.
+        /// </summary>
+        public JsonString(string value) => Value = value;
```

Comment:
So `JsonString()` uses `""` but `JsonString(string)` allows null to be passed; is that desirable?  Should this throw for null?

Reply:
Right, it shouldn't accept null as for such cases we would use null `JsonNode` (or `JsonNull` if we decide to bring it back).